DBM-Icecrown/FrostwingHalls/Valithria.lua

```diff
@@ -128,11 +128,11 @@ function mod:OnCombatStart(delay)
 	self:ScheduleMethod(46.5, "Portals")--This will never be perfect, since it's never same. 45-48sec variations
 	self.vb.BlazingSkeletonTimer = 60
 	self.vb.AbomTimer = 60
-	self.vb.AbomSpawn = 0
+	self.vb.AbomSpawn = 1
```

Comment:
```suggestion
	self.vb.AbomSpawn = 0
```
Keep it at 0, and add +1 to the count arg below on timerAbom